Makefile

```diff
@@ -1,9 +1,12 @@
 # Makefile for Sphinx documentation
-#
+
+SHELL=bash
```

Comment:
Is this required? I use zsh.

Reply:
I'll remove it, but even if you use zsh (I use fish), it's only for the variable handling internal to the processing of this Makefile. See https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html. I think we can remove it, I've added it thinking that it may be needed for the realpath part.

Reply:
Thank you for the reference. After reading that link, I think keeping the `SHELL` var is a good idea.

Reply:
This was resolved in e29c54136c1e9d75811c7a3fb2b344f10b635eaf